web-local/src/lib/components/workspace/metrics-def/MetricsDefTimeColumnSelector.svelte

```diff
@@ -70,4 +90,36 @@
       </TooltipContent>
     </Tooltip>
   </div>
+  {#if timeColumnSelectedValue !== "__DEFAULT_VALUE__"}
+    <Tooltip location="bottom" distance={8}>
+      <IconButton
+        compact
+        marginClasses="ml-1"
+        on:click={() => {
+          removeTimeseries();
+        }}
+      >
+        <CrossIcon color="gray" size="12px" />
+      </IconButton>
+      <TooltipContent slot="tooltip-content">
+        Remove timestamp column
+      </TooltipContent>
+    </Tooltip>
+  {:else}
+    <Tooltip location="bottom" distance={8}>
+      <IconButton
+        compact
+        marginClasses="ml-1"
+        on:click={() => {
+          noTimeseriesCTA();
+        }}
+      >
+        <AlertTriangle color="gray" size="16px" />
+      </IconButton>
+      <TooltipContent slot="tooltip-content">
```

Comment:
```suggestion
      <TooltipContent slot="tooltip-content" maxWidth="300px">
```

Looks a little funky without this.